core/src/main/scala/org/apache/spark/scheduler/SchedulerBackend.scala

```diff
@@ -41,4 +41,11 @@ private[spark] trait SchedulerBackend {
    */
   def applicationId(): String = appId
 
+  /**
+   * Get an application ID associated with the job.
+   *
```

Comment:
might be worth a comment even though that is the case the developer doesn't need to guess.